src/components/shared/Shared.js

```diff
@@ -0,0 +1,116 @@
+import instagramLogo from "../../assets/img/svg/instafram-logo.svg";
+import youtubeLogo from "../../assets/img/svg/youtube-logo.svg";
+import twitterLogo from "../../assets/img/svg/twitter-logo.svg";
+import facebookLogo from "../../assets/img/svg/facebook-logo.svg";
+import { Link } from "react-router-dom";
+import "./Shared.css";
+import { getSharedFolderSample, getSharedUserSample } from "../../api";
+import { useEffect, useState } from "react";
+import SharedMain from "./SharedMain";
+
+function Shared() {
+  const [userProfile, setUserProfile] = useState(null);
+  const [ownerProfile, setOwnerProfile] = useState(null);
+
+  const handleSharedUserProfile = async () => {
+    let result;
+
+    result = await getSharedUserSample();
```

Comment:
```suggestion
    const result = await getSharedUserSample();
```

result 를 바꾸는 부분이 이후에 없어서 const 에 담아주는 게 나을 거 같아요!

Reply:
네 수정하겠습니다~